src/LaunchDarkly.EventSource/Configuration.cs

```diff
@@ -129,114 +121,59 @@ public sealed class Configuration
         /// <seealso cref="ConfigurationBuilder.PreferDataAsUtf8Bytes(bool)"/>
         public bool PreferDataAsUtf8Bytes { get; }
 
+        /// <summary>
+        /// The timeout when reading from the EventSource API.
+        /// </summary>
+        /// <seealso cref="ConfigurationBuilder.ReadTimeout(TimeSpan)"/>
+        public TimeSpan ReadTimeout { get; }
+
         /// <summary>
         /// A factory for HTTP request body content, if the HTTP method is one that allows a request body.
         /// is one that allows a request body. This is in the form of a factory function because the request
         /// may need to be sent more than once.
         /// </summary>
-        /// <seealso cref="ConfigurationBuilder.RequestBodyFactory(HttpContentFactory)"/>
-        public HttpContentFactory RequestBodyFactory { get; }
+        /// <seealso cref="ConfigurationBuilder.RequestBodyFactory(Func{HttpContent})"/>
+        public Func<HttpContent> RequestBodyFactory { get; }
 
         /// <summary>
-        /// Gets the maximum amount of time to wait before attempting to reconnect to the EventSource API. 
-        /// This value is read-only and cannot be set.
+        /// The request headers to be sent with each EventSource HTTP request.
+        /// </summary>
+        /// <seealso cref="ConfigurationBuilder.RequestHeader(string, string)"/>
+        /// <seealso cref="ConfigurationBuilder.RequestHeaders(IDictionary{string, string})"/>
+        public IDictionary<string, string> RequestHeaders { get; }
+
+        /// <summary>
+        /// Gets the <see cref="System.Uri"/> used when connecting to an EventSource API.
         /// </summary>
         /// <value>
-        /// The maximum duration of the retry.
+        /// The <see cref="System.Uri"/>.
```

Comment:
This seems to be the only property here with a `<value>` tag. As I mentioned in another PR, I'm not entirely sure how important these are for the code doc views (generated HTML or in IDE), but maybe we could be more consistent with these.

Reply:
We had a bunch of them, I took most of them out, I missed this one and will take it out now.